docs/01-overview/README.md

```diff
@@ -8,9 +8,22 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
 ![overview](./assets/kyma-overview.svg)
 
+## Modular Kyma
+
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.
+
+The modular approach boosts Kyma's extensibility and scalability, making it even more cost-effective. With fully modularized Kyma, you can choose from many of its modules to facilitate your business needs best.  You can enable modules on demand and configure them. By disabling the modules you don't need to use, you considerably reduce the attack surface area, as well as the upgrade time and risk.
+
+### Kyma modules
+
+The first available Kyma modules are:
```

Comment:
We say there are some modules.  Below particular functionalities are mentioned. How does the module differ from the functionality described below ? Maybe we should say it differently and just list all functionalities and say some of them are available in core Kyma, and some as module.

Reply:
We have an explicit mention that the modules provide the functionality above: "Kyma components become modules, each providing one functionality developed independently"

Reply:
For me, this section is to explicitly link to the relevant documentation for the modules. not to convey functionality. This has been a major requirement since the beginning - provide a table with the modules and links to their documentation.